### ISSUE TITLE
Adjust unit categories to CLDR version

### ISSUE THREAD

**Issue by ccprog** [globalizejs/globalize#843]
Currently, the unit formatter uses a [hard-coded list of unit categories](https://github.com/globalizejs/globalize/blob/master/src/unit/categories.js). This list is getting longer by CLDR version. Currently (v33), the following entries are missing:

    "concentr", "consumption", "electric", "energy", "frequency"

See https://www.unicode.org/repos/cldr/tags/release-33/common/validity/unit.xml

**Comment by rxaviers** [comment]
True. We need to evaluate the feasibility/cost to using a list generated dynamically...
